recipes/scalene/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set package_name = "scalene" %}
+{% set version = "conda_2022_08_07" %}
```

Comment:
The previous tag was [v1.5.9.1](https://github.com/plasma-umass/scalene/releases/tag/v1.5.9.1). What was the reason for this change? It is quite unusual version  string. You can either use [SemVer](https://semver.org/) or [CalVer](https://calver.org/) in conda-forge. If you need to use that you should at least drop the `conda_` from the version string and make that a full CalVer replacing the `_` and `.`.

Reply:
Right am curious was this just done for testing purposes (to grab currently unreleased changes for testing)? If so (assuming we are happy with other things here), does it make sense to do a proper release that could then be used in the recipe?

Reply:
I do normally use SemVer. The version in the meta tag was indeed for testing. I am happy to issue a new release of Scalene (which would be 1.5.10). Just let me know if that's what needed, and I will do it.

One question I have is what the process is for issuing new releases on Conda; is it a new PR every time (like this one), or is there a more streamlined possibility going forwards?

Reply:
Once the PR is merged a feedstock will be created, see https://conda-forge.org/docs/maintainer/adding_pkgs.html#feedstock-repository-structure

If you follow SemVer, the bots will find the new tags and send PRs to the feedstock to update it. If there are no big changes you can usually just merge that one. Sometimes  you may need to adjust the dependencies.

Reply:
Sounds great. Please let me know if I need to issue a new release to get this merged.

Reply:
Yeah cutting a new release would be great. Seems like things are working otherwise

Reply:
On it.